src/blueapi/config.py

```diff
@@ -162,6 +162,9 @@ class OIDCConfig(BlueapiBaseModel):
     )
     client_id: str = Field(description="Client ID")
     client_audience: str = Field(description="Client Audience(s)", default="blueapi")
+    logout_redirect_endpoint: str | None = Field(
```

Comment:
```suggestion
    logout_redirect_endpoint: str | SkipJsonSchema[None] = Field(
```
Will this cause the apischema.yaml to not have

```yaml
        logout_redirect_endpoint:
          anyOf:
          - type: string
          - type: 'null'
```

the string or null should be captured in the required fields which is already correct